### PR TITLE
[sdl] Add support arrow key for next/prev page

### PR DESCRIPTION
**Pull request by robert00s**
Close: #4997 
This PR add support arrow key and mouse scroll wheel for next/previous page.

- Document in page mode:

PageDown, Arrow down, Arrow right - go to next page
PageUp, Arrow up, arrow left - go to prev page
Mouse scroll wheel down - go to next page
Mouse scroll wheel up - go to prev page

- Document in scroll mode:

PageDown, Arrow right - go to next page
PageUp, Arrow, arrow left - go to prev page
Arrow down - scroll document down
Arrow up - scroll document up
Mouse scroll wheel down - scroll document down
Mouse scroll wheel up - scroll document up

### CODE REVIEW

File: frontend/apps/reader/modules/readerpaging.lua [hunk 1]
```diff
@@ -396,8 +405,13 @@ function ReaderPaging:onPan(_, ges)
         if self.ui.gesture and self.ui.gesture.multiswipes_enabled then
             relative_type = "relative_delayed"
         end
-        self:onPanningRel(self.last_pan_relative_y - ges[relative_type].y)
-        self.last_pan_relative_y = ges[relative_type].y
+        --this is only use when mouse wheel is used
```
Comment (Frenzie) [inline frontend/apps/reader/modules/readerpaging.lua]: Sorry, I'm just slightly distracted by it. :-P
```suggestion
        -- this is only used when mouse wheel is used
```

File: frontend/apps/reader/modules/readerpaging.lua [hunk 2]
```diff
@@ -43,13 +43,22 @@ function ReaderPaging:init()
     self.key_events = {}
     if Device:hasKeys() then
         self.key_events.GotoNextPage = {
-            {Input.group.PgFwd}, doc = "go to next page",
+            { {"RPgFwd", "LPgFwd", "Right" } }, doc = "go to next page",
```
Comment (NiLuJe) [inline frontend/apps/reader/modules/readerpaging.lua]: Was there an issue with simply adding Left/Right to the PgBack/PgFwd input groups?

Comment (robert00s) [reply frontend/apps/reader/modules/readerpaging.lua]: input.group.PgFwd/PgBack is also used in another places eg. scrolltextwidget, keyvaluepage etc.

Comment (NiLuJe) [reply frontend/apps/reader/modules/readerpaging.lua]: Okay, that's what I thought, thanks for the confirmation ;).

**Pull request merged**